Makefile

```diff
@@ -166,8 +166,8 @@ $(GOLANGCI_LINT): ## Build golangci-lint from tools folder.
 $(KUSTOMIZE): ## Build kustomize from tools folder.
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) sigs.k8s.io/kustomize/kustomize/v3@v3.5.4
 
-$(MOCKGEN): ## Build mockgen from tools folder.
-	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) github.com/golang/mock/mockgen@v1.4.3
+$(MOCKGEN): ## Build mockgen from tools folder. Using unreleased version until https://github.com/golang/mock/pull/405 is part of a release.
```

Comment:
will switch to upcoming release once it's out https://github.com/golang/mock/pull/405#issuecomment-648449558